kiwi_keg/image_definition.py

```diff
@@ -136,3 +136,16 @@ def update_contents(self, include_paths):
                     contents
                 )
             self.data['contents'].update(contents)
+
+    def list_recipes(self):
+        images_recipes = []
+        images_files = KegUtils.get_all_files(self.image_root)
+
+        for image_file in images_files:
+            if os.path.basename(image_file) == 'image.yaml':
+                rel_path = os.path.relpath(
+                    os.path.dirname(image_file),
+                    self.image_root
+                )
+                images_recipes.append(rel_path)
+        return images_recipes
```

Comment:
OK, our list returns something that looks like:

`images/sles-sap/ondemand/15-sp2`

but when I want to create an image description what is expected is

`keg -r foo -d bar sles-sap/ondemand/15-sp2`

at least according to the help, because it states:

`SOURCE    Path to image source, expected under RECIPES_ROOT/images`

That means a user that does a list cannot use the output directly in the image description creation command. That's not nice.

So we should either fix the listing output or we should modify the way `SOURCE` is processed.